src/main.js

```diff
@@ -46,8 +51,31 @@ function newActivity() {
   buttonSection.classList.add('hidden');
   timerView.classList.remove('hidden');
   var category = currentCategory;
-  var description = document.querySelector('#description-input').value;
-  var minutes = document.querySelector('#minutes-input').value;
-  var seconds = document.querySelector('#seconds-input').value;
-  currentActivity = new Activity(category, description, minutes, seconds);
+  currentActivity = new Activity(category, descInput.value, minInput.value, secInput.value);
```

Comment:
This change makes perfect sense! We are no able to place these as global variables and use them in multiple functions.